src/services/mockBackend.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios'
+import AxiosMockAdapter from 'axios-mock-adapter'
+
+import mockData from './mockData'
+
+const moxios = axios.create()
+
+const axiosMockAdapterInstance = new AxiosMockAdapter(moxios, {
+  delayResponse: 0
+})
+
+const { notifications, patientRecord, goldenRecords } = mockData
+
+axiosMockAdapterInstance
+  .onGet('/MatchesForReview')
```

Comment:
```suggestion
  .onGet(ROUTES.GET_NOTIFICATIONS)
```
How about we re-use ROUTES

Reply:
Yeah, I was thinking about that actually. I'll put out a const class